packages/@aws-cdk/aws-ecs/test/test.l3s.ts

```diff
@@ -47,6 +47,55 @@ export = {
               Value: "test environment variable 2 value"
             }
           ],
+          Memory: 1024
+        }
+      ]
+    }));
+
+    test.done();
+  },
+
+  'test ECS loadbalanced construct with memoryReservationMiB'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const vpc = new ec2.VpcNetwork(stack, 'VPC');
+    const cluster = new ecs.Cluster(stack, 'Cluster', { vpc });
+    cluster.addCapacity('DefaultAutoScalingGroup', { instanceType: new ec2.InstanceType('t2.micro') });
+
+    // WHEN
+    new ecs.LoadBalancedEc2Service(stack, 'Service', {
+      cluster,
+      memoryReservationMiB: 1024,
+      image: ecs.ContainerImage.fromRegistry('test'),
+      desiredCount: 2,
+      environment: {
+        TEST_ENVIRONMENT_VARIABLE1: "test environment variable 1 value",
+        TEST_ENVIRONMENT_VARIABLE2: "test environment variable 2 value"
+      }
```

Comment:
Since the purpose of the test is just to ensure `memoryReservationMiB` is being set properly, I would  remove any other optional props from the constructor to make it clearer (e.g. `desiredCount` and `environment`).

Reply:
Updated